### DISCUSSION TITLE
全屏看视频

### DISCUSSION THREAD

**Discussion by Shine-On-You** [the1812/Bilibili-Evolved#1907] [功能建议 / Ideas]
能否做一个全屏看视频时（右上角或其他地方，）显示正常时间，以及全屏视频结束后自动退出全屏。谢谢

**Comment by FoundTheWOUT** [comment]
你好，你有使用过该插件的`v2`版本吗。如果你使用过，可尝试使用一下这个[组件](https://raw.githubusercontent.com/FoundTheWOUT/Bilibili-Evolved/v2/registry/dist/components/video/player/localtime.js)。安装方式可以看一下该项目的`v2`分支

**Comment by Geeyun-JY3** [comment]
全屏视频结束后自动退出全屏见：#655
